src/lib/Components/Inbox/Conversations/__tests__/Message-tests.tsx

```diff
@@ -15,6 +15,7 @@ it("looks correct when rendered", () => {
     created_at: moment().subtract(30, "minutes").toISOString(),
```

Comment:
Is this the test you were asking about? Because this is going to be flakey, me thinks. Better to mock the whole module out.